### DISCUSSION TITLE
Welcome fellow pleebs

### DISCUSSION THREAD

**Discussion by JayMartin05** [JustinRoberts5760/convoy-sGameOfLife#6] [General]
This is general discussion if you have ideas or more specific questions please open a new disccusion

**Comment by JustinRoberts5760** [comment]
cheers

**Comment by alikahwaji** [comment]
@JayMartin05 since @JustinRoberts5760  setup the repo, he will be the only one to access the settings :)

**Comment by JayMartin05** [comment]
Hey all
Lets keep communications here and away from teams please. 
Make sure you work on your CSA assesment tonight first. Then look into your tasks (Issues for now) come back tomorrow at lunch time with where you are at with your tasks and we can reaccess what needs to be done over the weekend.

**Comment by JustinRoberts5760** [comment]
I have invited all to a Kanban where tasks will be posted for each member from @JayMartin05
